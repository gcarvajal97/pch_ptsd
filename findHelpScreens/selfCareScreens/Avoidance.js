import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView,} from 'react-native';
import translate from '../../components/translateService';

class Avoidance extends Component {

  render() {
    return (
        <View>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.avoidance.paragraphTitle1')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph1')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph2')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.avoidance.paragraphTitle2')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph3')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph4')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.avoidance.paragraphTitle3')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph5')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph6')}</Text>
            <Text style={styles.paragraphTitle}>{translate('selfCare.content.avoidance.paragraphTitle4')}</Text>
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.why')}</Text>
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph7')}</Text> 
            <Text style={styles.paragraphBold}>{translate('selfCare.content.avoidance.problem')}</Text>  
            <Text style={styles.paragraph}>{translate('selfCare.content.avoidance.paragraph8')}</Text>                
        </View>
    );
  }
}

export default Avoidance;

const styles = StyleSheet.create({
    paragraphTitle: {
      fontWeight: 'bold',
      fontSize: 20,
      alignSelf: 'stretch',
      textAlign: 'left',
      marginTop: 6,
      marginBottom: 3,
      color: '#2089DC'
    },
    paragraph: {
      textAlign: 'left',
      fontSize: 14,
      marginVertical: 2,
    },
    paragraphBold: {
      textAlign: 'left',
      fontSize: 14,
      fontWeight: 'bold',
      marginVertical: 2,
      marginLeft: 10
    }
  });