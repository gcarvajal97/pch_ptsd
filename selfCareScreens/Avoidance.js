import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView,} from 'react-native';

class Avoidance extends Component {

  render() {
    return (
            <Text>

                <Text style={styles.title}>
                    AVOIDANCE SYMPTOMS{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I don't want to talk or think about what happened."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}Everyone needs to take a break sometimes from thinking about or 
                    talking about a scary experience. 
                    or traumatic event.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Do you never want to think or talk about it with anyone, 
                    or does talking about it make you feel very upset?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I want to stay away from people, places, or things that are reminders of what happened."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}After something scary happens, reminders of what happened can make us feel 
                    very anxious, and we might start to avoid those things.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Is avoiding reminders stopping you from doing things you think you 
                    should be able to do, or from enjoying activities even if you force yourself to do them?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I have lost interest in activities I usually enjoy, since the injury."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}One way we cope with overwhelming scary experiences is to 
                    "shut down" our emotions for a while.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Are there lots of thing you used to enjoy that you 
                    just don't feel like dong now? Are you feeling emotionally "numb" much of the time?{"\n"}
                </Text>

                <Text style={styles.subTitle}>
                    {"\n"}"I just don't feel like being with people, since the injury."{"\n"}
                </Text>
                <Text style={styles.paragraph}>
                    {"\n"}Why it happens: {"\n"}One way we cope with overwhelming scary experiences is to "shut down" 
                    our emotions and not feel close to other people for a while.{"\n"}
                    {"\n"}Is this a problem?: {"\n"}Are you having a really hard time feeling relaxed and close to 
                    friends and family you usually trust and like to spend time with?
                </Text>                
                
            </Text>
    );
  }
}

export default Avoidance;

const styles = StyleSheet.create({
  container: {
      flex: 1,
      alignItems: 'center',
      margin: 10,
      backgroundColor: 'white',
      shadowColor: "#000",
      shadowOffset: {
          width: 0,
          height: 2,
      },
      shadowOpacity: 0.23,
      shadowRadius: 2.62,
      elevation: 4,
  },
  title: {
      backgroundColor: 'white', 
      color: '#2089DC', 
      fontWeight: 'bold', 
      fontSize: 22, 
      textAlign: 'left', 
      alignSelf: 'stretch'
  },
  subTitle: {
    backgroundColor: 'white', 
    color: '#2089DC', 
    alignSelf: 'stretch', 
    paddingLeft: 15, 
    paddingRight:15, 
    fontSize: 15, 
    borderBottomWidth:2, 
    borderBottomColor:'#2089DC',
  },
  paragraph: {
      alignSelf:"center", 
      paddingVertical:1, 
      paddingHorizontal:5,
      fontSize: 14,
      marginTop: 5,
  },
});