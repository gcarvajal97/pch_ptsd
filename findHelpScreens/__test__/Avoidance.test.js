import React from 'react';
import Avoidance from '../selfCareScreens/Avoidance';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<Avoidance/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 