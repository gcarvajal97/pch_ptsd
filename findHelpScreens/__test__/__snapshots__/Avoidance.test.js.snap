// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I don't want to talk or think about what happened."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Everyone needs to take a break sometimes from thinking about or talking about a scary experience or traumatic event.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Do you never want to think or talk about it with anyone, or does talking about it make you feel very upset?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I want to stay away from people, places, or things that are reminders of what happened."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    After something scary happens, reminders of what happened can make us feel very anxious, and we might start to avoid those things.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is avoiding reminders stopping you from doing things you think you should be able to do, or from enjoying activities even if you force yourself to do them?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I have lost interest in activities I usually enjoy, since the injury."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    One way we cope with overwhelming scary experiences is to "shut down" our emotions for a while.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Are there lots of thing you used to enjoy that you just don't feel like dong now? Are you feeling emotionally "numb" much of the time?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    "I just don't feel like being with people, since the injury."
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Why it happens:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    One way we cope with overwhelming scary experiences is to "shut down" our emotions and not feel close to other people for a while.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "fontWeight": "bold",
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Is this a problem?:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Are you having a really hard time feeling relaxed and close to friends and family you usually trust and like to spend time with?
  </Text>
</View>
`;
